FeatureTest.md

```diff
@@ -1,3 +1,18 @@
+# Motivating Scenarios
+1. In MVP + 1 (the version of the WebAssembly [after MVP](PostMVP.md)), [`i32.min_s`](FutureFeatures.md#additional-integer-operations) is introduced. A WebAssembly developer updates their toolkit and targets the new features. The compiler leverages `i32.min_s` when compiling a module the developer wrote. The developer's WebAssembly module works correctly both on execution environments at MVP, as well as those updated on MVP + 1.
+
+2. In MVP + 1, module authors may now use [Threading](PostMVP.md#threads) APIs in the browser. A developer wants to leverage multithreading in their module.
+  * In one variant of the scenario, our developer does not want to pay the engineering cost of developing and supporting a threaded and non-threaded version of their code. They opt not to support MVP targets, and only support MVP + 1 targets. End-users (browser users) get some message indicating they need MVP support.
+  * In another variant, our developer explicitly authors both MVP-only and MVP + 1 (with threads) code.
```

Comment:
I see. I updated referencing PostMVP as such, but not assuming anything about whether there will or not be a coarser grained "WebAssembly v2" milestone, or just finer grained features. The scenarios should help us tease apart what makes sense here.